Documentation/docs/the-model.md

```diff
@@ -8,10 +8,10 @@ The two dimensional domain to be meshed can be bounded by at most one exterior b
 
 If no model block is included at all, then a purely Cartesian mesh will be created using parameters set in the control file.
 
-As an example, the model in Fig. 16 has one outer boundary - the outer triangle - and three inner boundaries whose interiors are exterior to the mesh.
+As an example, the model in Fig. 16 has one outer boundary - the outer triangle - and four inner boundaries whose interiors are exterior to the mesh.
 
-![AllFeatures](https://user-images.githubusercontent.com/3637659/121807794-f3d45f00-cc55-11eb-9284-af5f4eed2c87.png)
-<p align = "center"> Fig. 16. A mesh whose model uses all curve types. Three `END_POINTS_LINE`s for the outer triangle. A `SPLINE_CURVE` for the free-form inner boundary, and circles defined by a `PARAMETRIC_EQUATION_CURVE` and by a `CIRCULAR_ARC` curve (<em>Examples/2D/AllFeatures</em>).</p>
+![AllFeatures](images/AllFeatures.png)
```

Comment:
```suggestion
![AllFeatures](https://github.com/user-attachments/assets/2f12ca1c-8a20-4c7d-98e6-9d619562a34d)
```
This is to avoid adding larger image files to the `docs` folder and have github "host" our documentation figures. It is an imperfect solution, but it works :)